artiq/gateware/rtio/phy/urukul_monitor.py

```diff
@@ -0,0 +1,141 @@
+from migen import *
+
+from artiq.coredevice.ad9910 import AD9910_REG_FTW, AD9910_REG_PROFILE0, AD9910_REG_PROFILE7
+from artiq.coredevice.ad9912_reg import AD9912_FTW0, AD9912_POW1
+from artiq.coredevice.spi2 import SPI_CONFIG_ADDR, SPI_DATA_ADDR, SPI_END
+from artiq.coredevice.urukul import CS_DDS_CH0, CS_DDS_MULTI, CS_CFG, CFG_IO_UPDATE
+
+
+class _AD9910_AD9912MonitorGeneric(Module):
```

Comment:
AD991x

Reply:
Not a good name since this mean AD9914 is part of it

Reply:
In a file named urukul_monitor it isn't.

Reply:
okay, let's split it into two files then

Reply:
There's nothing wrong with urukul_monitor.py... _AD991xMonitor in urukul_monitor.py is fine, short and not ambiguous.

Reply:
done